code/game/area/areas.dm

```diff
@@ -201,32 +201,34 @@ GLOBAL_LIST_EMPTY(teleportlocs)
   * Sends to all ai players, alert consoles, drones and alarm monitor programs in the world
   */
 /area/proc/poweralert(state, obj/source)
+	if (area_flags & NO_ALERTS)
+		return
 	if (state != poweralm)
 		poweralm = state
 		if(istype(source))	//Only report power alarms on the z-level where the source is located.
 			for (var/item in GLOB.silicon_mobs)
 				var/mob/living/silicon/aiPlayer = item
-				if (state == 1)
+				if (!state)
```

Comment:
```suggestion
				if (state)
```

Reply:
This is correct. I flipped the state var around because it made no sense before (state = TRUE meant stop alerting).